motionplan/cBiRRT.go

```diff
@@ -198,10 +198,23 @@ func (mp *cBiRRTMotionPlanner) rrtBackgroundRunner(
 		default:
 		}
 
-		tryExtend := func(target []referenceframe.Input) (node, node, float64) {
+		tryExtend := func(target []referenceframe.Input) (node, node, float64, error) {
```

Comment:
Returning 4 values here seems less than ideal.  I'm wondering if we need to return `reachedDelta` here.  Can we instead just return the nodes and if the algorithm needs to compute this after the fact then it always can.

Reply:
You're right; we don't need to be computing the delta in here. Will update.